governance/votes/Executive vote - November 19, 2021.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Template - [Executive Proposal] Onboarding WBTC-B, Parameter Changes, Change of Covenants for P1-DROP, Offboarding Collaterals - November 19, 2021
+summary: Onboarding WBTC-B; parameter changes; change of covenants for P1-DROP; offboarding BAL-A, COMP-A, AAVE-A, LRC-A, BAT-A, ZRX-A, UNIV2LINKETH-A
+date: 2021-11-19T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Onboarding WBTC-B, Parameter Changes, Change of Covenants for P1-DROP, Offboarding Collaterals - November 19, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- Increase to the WBTC-A Stability Fee.
```

Comment:
So when the details can fit into one line it's alright to put them in I would write this first one:
"- The WBTC-A [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) will be increased from 2.5% to 4.0%."